Data Science TOC first iteration

tanhakabir opened the pull request:
Changes:
1. Add Data Science overview page for the Data Science topic
2. Moves Jupyter Notebook content and related images into the Data Science topic. (No actual content was changed, it's still webview version, I just a dragged and dropped)
2. Moves Data Science Tutorial content and related images into the Data Science topic. (No actual content was changed, I just a dragged and dropped)
3. Remove the Jupyter and Data Science tutorial articles under python and replaced them with just "Data Science" to point to the Data Science overview page.
3. Any shared images were copied over into the Data Science TOC
4. An empty page for Python Interactive to link to the Python topic's Python Interactive

**Important:** Need to merge in https://github.com/microsoft/vscode-website/pull/973 beforehand for this to work

tanhakabir commented:
Here's a gif of what this looks like:
![recording (44)](https://user-images.githubusercontent.com/12758612/119695186-ca68a600-be02-11eb-80f3-7cd5eb98b1d1.gif)

JoshuaPartlow commented:
Hi @tanhakabir / @claudiaregio - Looking at this PR I think my one question would be whether moving the Jupyter support topic, that will eventually be the new native notebooks topic (I assume), accounts for the fact that we'll still have some notebooks features that are Python specific. Would the plan be, when Native Notebooks GAs, to update the Data Science Notebook support page and instead of doing a ToC link in the Python node, have a Python specific notebook support topic for Python that points to the Data Science node for general info and then talks about Python specific features in Notebooks?

+ @luabud and @jmew as an fyi about these changes.

tanhakabir commented:
@luabud @jmew tagging for your feedback as well!

tanhakabir commented:
Added Data Science overview page was added highlighting .NET notebook:
![image](https://user-images.githubusercontent.com/12758612/119891033-62db5500-beed-11eb-8160-cfb285706569.png)

Removed Jupyter and Data Science Tutorial underneath Python and instead replaced both with just "Data Science" that links to the Overview page in the Data Science TOC.

![recording (45)](https://user-images.githubusercontent.com/12758612/119891311-acc43b00-beed-11eb-88d9-76e1f964a701.gif)

claudiaregio commented:
I took a look at other docs, think we should change "What else can I do with notebooks?" to "What else can you do with notebooks?". The other docs tend to say you/your

jmew commented:
Discussed offline with @claudiaregio and for the PyTorch article https://code.visualstudio.com/docs/python/pytorch-support, the plan would be to move it into the Data Science node, but have the old link redirect to the new page link such that our existing evangelism of the link doesnt go to waste.

tanhakabir commented:
@jmew will do in another PR after this one gets merged in

luabud commented on docs/datascience/overview.md:
```diff
@@ -0,0 +1,29 @@
+
+The VS Code Marketplace offers a family of extensions that provide a first-class data science experience for Python data science. In order to get started with Python, Microsoft strongly recommends the following extensions:
+
+<div class="marketplace-extensions-datascience-python"></div>
```
@tanhakabir I'm most likely wrong but figured I'd ask just in case hehe 
is it missing an extension list here by any chance?

tanhakabir replied:
Oh yeah sorry this list is on a different repo, the vscode-website repo but you can see the list I have in the screenshots above

luabud replied:
Cool, thank you!! 😊
